### PR TITLE
New Nukie Song "Last Call"

### PR DESCRIPTION
**Pull request by he1acdvv**
### This is my first PR that i've done single-handedly. I'm sincerely sorry if i got something wrong. 

      ❗ ❗ ❗ 

## About the PR

Added a new Nuke Detonation Song. (again)

## Why / Balance

I felt like making a new nuke ost again.

## Technical details

I went to the wizards of the forest, who gave me divine inspiration.

## Media

https://github.com/user-attachments/assets/01b43281-1f01-45fd-ba56-3fb9583f151a

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

:cl: mryikes
- add: New Nuke Detonation Song "Last Call".

### CODE REVIEW

**Comment by he1acdvv** [comment]
> Approval just indicates it is implemented correctly. Good job on your first PR by yourself.

phew.. Thank you! : D

File: Resources/Audio/StationEvents/last_call.ogg
Comment (he1acdvv) [inline Resources/Audio/StationEvents/last_call.ogg:1]: yeah no i'm sorry, it should be working fine now.

**Comment by baddiePRO** [comment]
album cover goes hard

**Comment by he1acdvv** [comment]
> album cover goes hard

made it myself

**Comment by XxSWAG-MASTER-OVERLORDxX** [comment]
bro cooked, and its straight fire.

**Comment by ps3moira** [comment]
After relistening to all 4 nukie songs and listening to this, this song doesn't fit on the same level as the other songs we have. Especially at 0:40 it just doesn't fit and sounds like it was meant for a different game.

**Comment by he1acdvv** [comment]
> After relistening to all 4 nukie songs and listening to this, this song doesn't fit on the same level as the other songs we have. Especially at 0:40 it just doesn't fit and sounds like it was meant for a different game.

Not all of the songs are supposed to sound alike. I tried to bring a little bit of a new sound into the circle. 
After all, it's really a matter of personal preference, but it's never a bad thing to try new stuff, is it? Some change here and there can be good.

**Comment by ps3moira** [comment]
> > After relistening to all 4 nukie songs and listening to this, this song doesn't fit on the same level as the other songs we have. Especially at 0:40 it just doesn't fit and sounds like it was meant for a different game.
> 
> Not all of the songs are supposed to sound alike. I tried to bring a little bit of a new sound into the circle. After all, it's really a matter of personal preference, but it's never a bad thing to try new stuff, is it? Some change here and there can be good.

The point of nuke countdown music is that it sounds like a countdown and that there's a rush to disarm, this song however doesn't do that all that well (but I can see the hints of elements inspired from other songs) and sounds more like a space invaders type of song. I want to hammer home that this isn't unsalvageable and it's sound can be improved upon to fit the theme of nukies going on station murking crew, the nuke counting down and the station sounding like it's going to blow up.

**Comment by he1acdvv** [comment]
> > > After relistening to all 4 nukie songs and listening to this, this song doesn't fit on the same level as the other songs we have. Especially at 0:40 it just doesn't fit and sounds like it was meant for a different game.
> > 
> > 
> > Not all of the songs are supposed to sound alike. I tried to bring a little bit of a new sound into the circle. After all, it's really a matter of personal preference, but it's never a bad thing to try new stuff, is it? Some change here and there can be good.
> 
> The point of nuke countdown music is that it sounds like a countdown and that there's a rush to disarm, this song however doesn't do that all that well (but I can see the hints of elements inspired from other songs) and sounds more like a space invaders type of song. I want to hammer home that this isn't unsalvageable and it's sound can be improved upon to fit the theme of nukies going on station murking crew, the nuke counting down and the station sounding like it's going to blow up.

Well, there are still elements of that present, i just wanted for it to have a more hopeful feel. I've done the previous track "Clearly Nuclear" the way you just described it.

**Comment by he1acdvv** [comment]
Well, i personally do believe this can add to SS14's atmosphere. It's up to the people (and maintainers) to decide whether this goes in or not. If the community doesn't like that much, well.. We can always revert it.

**Comment by he1acdvv** [comment]
waiting patiently. 

image

**Comment by VasilisThePikachu** [comment]
Hey sorry for the delay. Unfortunately currently there is a freeze on new music of any kind until a streaming option is available. This freeze was done since music files increase the file size of the game by quite a bit (you can read more about frozen and restricted items on this issue #8524)

Granted the freeze page could have been better explaining that all music was frozen so I apologize for that.

You are free to request this PR to be reopened once the freeze prerequisite has been met. Once again I apologize.

**Comment by he1acdvv** [comment]
3 mb of files........

at least let's hope the solution won't take a year or so

**Comment by he1acdvv** [comment]
a bit bummed out since i did ask some people whether nuke songs fall under the music freeze and i was told - apparently not.

**Comment by a-person5660** [comment]
> 3 mb of files........
> 
> at least let's hope the solution won't take a year or so

Don't worry! The solution will be ready shortly after the heat death of the universe!

**Comment by Simyon264** [comment]
#29345 was merged as well by sloth. Do nukie songs count too? I'd really love more nuke detonation songs, as those events are kind of rare and not as big as lobby songs.

**Comment by VasilisThePikachu** [comment]
As far as I'm aware. We can't accept new songs of any kind due to the size they will take on the repo/the final game client. And in general as described in the freeze doc. "We have too many songs"

Maybe sloth missed it, or I'm missing out a point myself. I had tried asking for maintainer opinions about this pr and I got silence. So I'm working on my own discretion is a maintainer.

**Comment by Simyon264** [comment]
> I had tried asking for maintainer opinions about this pr and I got silence.

Could you try again? I would really love to see this in the game, especially because it is custom made.

**Comment by he1acdvv** [comment]
honestly -- don't merge this since i've made an improved version anyways.

i'll make a pr when the time's right

**Comment by VasilisThePikachu** [comment]
Heya, I this time got some opinions from some of the maintainer team and the conclusion is that this was correctly marked as frozen.

> https://github.com/space-wizards/space-station-14/pull/29345 was merged as well by sloth.
This is cause this was the music used for the trailer, and we thought it would be explainable for this reason.
Maintainers may skip over a freeze in certain cases. The song being used in the trailer and technically being the "Space Station 14" song gave it some reasons.

> 3 mb of files........
It may not seem a lot, but 3 mb can be quite a lot. Not everyone has an amazing internet. And also merging one song will result in a bunch of other pull requests being made and inflate the game of the final build of the game the players get served. We would like to keep file size as small as possible as of currently till we have better systems in place.

I apologize once again for this. You are more than welcome to make a new pull request once this freeze is lifted.

**Pull request closed without merging**